src/rust/ide/lib/view/src/graph_editor/component/visualization/container.rs

```diff
@@ -337,8 +352,8 @@ impl ContainerModel {
         } else {
             self.view.background.shape.radius.set(CORNER_RADIUS);
             self.view.overlay.shape.radius.set(CORNER_RADIUS);
-            self.view.background.shape.sprite.size.set(size);
-            self.view.overlay.shape.sprite.size.set(size);
+            self.view.background.shape.sprite.size.set(size.add_scalar(SHADOW_SIZE*2.0));
+            self.view.overlay.shape.sprite.size.set(size.add_scalar(SHADOW_SIZE*2.0));
```

Comment:
guidelines do actually want spaces aroung operators like `*`
(only `:` doesn't get them in some cases)